package.json

```diff
@@ -62,6 +62,7 @@
     "@babel/preset-typescript": "^7.23.3",
     "@babel/register": "^7.0.0",
     "@ebay/skin": "^17.0.0",
+    "@figma/code-connect": "^0.1.2",
```

Comment:
I think we need to also add a new script that we will need to use to make the connection

```
"scripts": {
   ...,
   "figma:connect": "figma connect publish \"https://...\" --token $FIGMA_TOKEN"
}
```

Reply:
@HenriqueLimas this has been added, thanks!